www/artworks/post.php

```diff
@@ -101,7 +101,7 @@ function parseArtworkTags(): array{
 
 	handleImageUpload($_FILES['color-upload']['tmp_name'], $artwork);
 
-	$_SESSION['success-message'] = '“' . $artwork->Name . '” submitted successfully!';
+	$_SESSION['success-message'] = '“' . Formatter::ToPlainText($artwork->Name) . '” submitted successfully!';
```

Comment:
In general we only want to escape at the point where we output to HTML, which is typically only within a `<?= ?>` block. Escaping elsewhere invites accidental double-escapes, and also can lead to a problem where further processing before output expects unescaped text, but the text was surprisingly escaped earlier in processing, possibly even in a different function.

Reply:
Good guideline about `<?= ?>` blocks, thanks. I changed the call to where this message is output to HTML.